.github/workflows/check-branch.yml

```diff
@@ -0,0 +1,34 @@
+name: PN Logging
+on: [push]
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+jobs:
+
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Set up Node.js'
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: .nvmrc
+          cache: 'yarn'
+      - name: 'Yarn Install'
+        run: yarn install --immutable
```

Comment:
Not at all a blocker to merging, just curious -- if we wanted to only install modules once we'd need to run lint and test on the same job?

Reply:
Yes, jobs are isolated to separate containers so any required setup must happen for each one. In this case, it might have made sense to keep them together, but in some of the larger projects like neodarwin the lint blocks testing for long enough that it makes sense to run them in parallel. Here I opted for consistency between projects since we've been using this patten elsewhere. From a timing perspective, it makes little difference because the setup happens in parallel, but we could potentially save credits by keeping lint/test combined for smaller projects and only run one container.